code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -35,5 +36,13 @@ Bonus
 				M.emote("sniff")
 			else
 				M.emote("sneeze")
-				A.spread(A.holder, 5)
-	return
\ No newline at end of file
+				Spreadzone(M, A)
+				var/obj/item/I = M.get_active_hand()
+				if(I)
+					M.drop_item()
+	return
+
+/datum/symptom/sneeze/proc/Spreadzone(mob/living/M, datum/disease/advance/A)
+	var/sneeze_zone = ((sqrt(21+A.totalResistance()))+(sqrt(21+A.totalTransmittable()))/2)

```

Comment:
oranges this is an add add divide you cannot go negative on this. That's what the offsets are for.

I'm not sure what you mean by too many Phil. I tried removing a few but I errored out on compiling, and I'm not sure what math system byond uses (PEMDAS or BODMAS).